src/Surfnet/StepupMiddleware/ApiBundle/Identity/Repository/RaCandidateRepository.php

```diff
@@ -215,9 +215,9 @@ public function createSearchQuery(RaCandidateQuery $query)
      */
     public function createOptionsQuery(RaCandidateQuery $query)
     {
-        $queryBuilder = $this->createQueryBuilder('rac')
-            ->select('rac.raInstitution')
-            ->groupBy('rac.raInstitution');
+        $queryBuilder = $this->createQueryBuilder('r')
+            ->select('r.institution, r.raInstitution')
```

Comment:
Isn't one of them currently abundant?

Reply:
Yes it is, my rationale was to leave it. But stripping it away might be a better idea. Fixing this.